web/src/engine/websites/Kakaopage.ts

```diff
@@ -0,0 +1,220 @@
+        const gql = `
+            query contentHomeOverview($seriesId: Long!) {
+              contentHomeOverview(seriesId: $seriesId) {
+                content {
+                  ...SeriesFragment
+                }
+              }
+            }
+            fragment SeriesFragment on Series {
+              id
+              seriesId
+              title
+            }
+        `;
+        const vars = { seriesId: id };
+        const request = new FetchRequest(new URL('/graphql', this.URI).href, {
+            headers: {
+                Referer: this.URI.href,
+                Origin: this.URI.href
+            }
+        });
+        const operationName = 'contentHomeOverview';
+        const { contentHomeOverview } = await FetchGraphQL<ApiClipBoard>(request, operationName, gql, JSON.stringify(vars));
+        return new Manga(this, provider, String(contentHomeOverview.content.seriesId), contentHomeOverview.content.title.trim());
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        let nextCursor = '';
+        const chapterList = [];
+        for (let run = true; run;) {
+            const data = await this.getChaptersFromPage(manga, nextCursor);
+            const chapters = data.contentHomeProductList.edges.map(chapter => new Chapter(this, manga, chapter.node.single.productId.toString(), chapter.node.single.title.replace(manga.Title, '').trim()));
+            nextCursor = data.contentHomeProductList.pageInfo.hasNextPage ? data.contentHomeProductList.pageInfo.endCursor : null;
+            chapterList.push(...chapters);
+            run = nextCursor != null;
+        }
+        return chapterList;
+    }
+
+    async getChaptersFromPage(manga: Manga, nextCursor: string): Promise<APIChapters> {
+        const gql = `
+            query contentHomeProductList(
+              $after: String
+              $before: String
+              $first: Int
+              $last: Int
+              $seriesId: Long!
+              $boughtOnly: Boolean
+              $sortType: String
+            ) {
+              contentHomeProductList(
+                seriesId: $seriesId
+                after: $after
+                before: $before
+                first: $first
+                last: $last
+                boughtOnly: $boughtOnly
+                sortType: $sortType
+              ) {
+                totalCount
+                pageInfo {
+                  hasNextPage
+                  endCursor
+                  hasPreviousPage
+                  startCursor
+                }
+                edges {
+                  cursor
+                  node {
+                    ...SingleListViewItem
+                  }
+                }
+              }
+            }
+            fragment SingleListViewItem on SingleListViewItem {
+              id
+              type
+              single {
+                productId
+                id
+                isFree
+                thumbnail
```

Comment:
Seems to me like some of the properties are not used by HakuNeko and can be removed